src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -64,10 +64,8 @@
                   persistent-hint
                   filled
         >
-          <template slot="item" slot-scope="data">
-          <span class="list-item">
-            {{ data.item.text }}
-          </span>
+          <template v-slot:item="data">
```

Comment:
This is how to combine the slot with the slot-scope. I tested this and it works.

The reason for this block is to format each data item in this list. Otherwise this block can be left out.